recipes-domd/agl/inc/domd-agl-image.inc

```diff
@@ -26,6 +26,9 @@ SRC_URI_append = " \
     git://git.yoctoproject.org/meta-virtualization;destsuffix=repo/meta-virtualization;branch=morty \
     git://git.yoctoproject.org/meta-selinux;destsuffix=repo/meta-selinux;branch=morty \
 "
+# LLVM Clang section. Needed to build gles-user-module
+XT_QUIRK_BB_ADD_LAYER += " meta-clang"
+SRC_URI_append = " git://github.com/kraj/meta-clang.git;destsuffix=repo/meta-clang;branch=krogoth"
```

Comment:
why krogoth? pls align with the rest which is morty

Reply:
Done. (tested with morty)